README.md

```diff
@@ -603,6 +624,19 @@ application log level is set to critical (1/-1) or a higher logging level. A cal
 to `Logger.LOG_EXCEPTION(Exception)` will result in log output that is visible if the configured
 application log level is set to trace (6/-6).
 
+# Cumulocity Inventory Objects Update
+For applications that want additional device information to be displayed in Cumulocity, the Ewon Flexy Cumulocity Connector supports the following inventory object update feature. 
+For all valid files found in the `/usr/CumulocityInventoryObjects` directory, the connector will read the file and update the inventory object in Cumulocity. The file should be in JSON format and have the `.json` extension.
+The file name, excluding the .json extension, will be used as the device ID for the update. A special file name `parent.json` is reserved for inventory object updates for the parent device (the Flexy). 
+All other file names will be used as the device ID for child device updates. 
+The connector does not complete any validation of the file contents. It is the responsibility of the user to ensure the file contents are valid JSON and follow the Cumulocity inventory object schema. 
+
+## Inventory Object Update Trigger
+Inventory object updates are triggered by a special tag named `CumulocityInventorUpdate`. Users that want to enable this feature should create this tag as a boolean type. To trigger an update, users should set the tag value to `1`. The tag value will be reset to `0` after the update is complete if one update was sent without error. For update failures are logged, but no further action is taken. Therefore users can retry the update by setting the tag value to `1` again. 
```

Comment:
```suggestion
Inventory object updates are triggered by a special tag named `CumulocityInventoryUpdate`. Users who want to enable this feature should create this tag as a boolean type. To trigger an update, users should set the tag value to `1`. The tag value will be reset to `0` after the update is complete if one update was sent without error. An event is logged for update failures, but no further action is taken. Therefore, users can retry the update by setting the tag value to `1` again. 
```

Spelling and minor changes. The second to last sentence also needs to be reworded as it does not read correctly in its current state.

Reply:
will change